### PR TITLE
[minor] Adding many more companies to powered by page

### PR DESCRIPTION
**Pull request by scott-confluent**

### CODE REVIEW

File: powered-by.html [hunk 1]
```diff
@@ -43,15 +43,30 @@
         "logoBgColor": "#ffffff",
         "description": "Apache Kafka powers the backbone of Agoda's data pipeline with trillions of events streaming through daily across multiple data centers. The majority of the events are destined for analytical systems and directly influence business decisions at one of the world’s fastest growing online travel booking platforms."
     }, {
-        "link": "http://www.amadeus.com",
-        "logo": "amadeus.jpg",
+        "link": "https://agoora.com/",
+        "logo": "spoud-logo.svg",
         "logoBgColor": "#ffffff",
-        "description": "Apache Kafka is used for both real-time and batch data processing, and is the chosen event log technology for Amadeus microservice-based streaming applications. Kafka is also used for operational use cases such as application logs collection."
+        "description": "We have a product, called AGOORA, which is a data-platform for Apache Kafka that encourages collaboration around data transparency and ownership."
```
Comment (vvcephei) [inline powered-by.html]:
```suggestion
        "description": "AGOORA is a data-platform for Apache Kafka that encourages collaboration around data transparency and ownership."
```

File: powered-by.html [hunk 2]
```diff
@@ -62,6 +77,11 @@
         "logo": "ants.png",
         "logoBgColor": "#ffffff",
         "description": "Ants.vn use Kafka in production for stream processing and log transfer (over 5B messages/month and growing)"
+    }, {
+        "link": "http://appsflyer.com/",
+        "logo": "ants.png",
+        "logoBgColor": "#ffffff",
+        "description": "Ants.vn use Kafka in production for stream processing and log transfer (over 5B messages/month and growing)"
```
Comment (vvcephei) [inline powered-by.html]:
```suggestion
        "description": "Ants.vn uses Kafka in production for stream processing and log transfer (over 5B messages/month and growing)"
```

File: powered-by.html [hunk 3]
```diff
@@ -237,6 +292,26 @@
         "logo": "ipinyou.png",
         "logoBgColor": "#ffffff",
         "description": "The largest DSP in China which has its HQ in Beijing and offices in Shanghai, Guangzhou, Silicon Valley and Seattle. Kafka clusters are the central data hub in iPinYou. All kinds of Internet display advertising data, such as bid/no-bid, impression, click, advertiser, conversion and etc., are collected as primary data streams into Kafka brokers in real time, by LogAggregator (a substitute for Apache Flume, which is implemented in C/C++ by iPinYou, has customized functionality, better performance, lower resource-consuming)."
+    }, {
+        "link": "https://www.ironsrc.com/",
+        "logo": "ironsource.png",
+        "logoBgColor": "#ffffff",
+        "description": "ironSource powers the growth of the world's top games, using Apache Kafka as the backbone infrastructure for the async messaging of millions of events per second that run through their industry-leading game growth platform. In addition ironSource uses the Kafka Stream API to handle multiple real-time use cases, such as budget management, monitoring and alerting."
```
Comment (vvcephei) [inline powered-by.html]:
```suggestion
        "description": "ironSource powers the growth of the world's top games, using Apache Kafka as the backbone infrastructure for the async messaging of millions of events per second that run through their industry-leading game growth platform. In addition ironSource uses the Kafka Streams API to handle multiple real-time use cases, such as budget management, monitoring and alerting."
```

File: powered-by.html [hunk 4]
```diff
@@ -237,6 +292,26 @@
...
+    }, {
+        "link": "http://banno.com",
+        "logo": "JHD_Logo.jpg",
+        "logoBgColor": "#ffffff",
+        "description": "The Banno Digital Platform from Jack Henry enables community financial institutions to provide world-class service in today’s digital landscape. The Banno team integrates various streams of data through Apache Kafka, reacting to events as they occur, to provide innovative banking solutions."
+    }, {
+        "link": "https://www.kuaishou.com/en",
+        "logo": "KuaishouLogo.png",
+        "logoBgColor": "#FFFFFF",
+        "description": "At kuaishou, Kafka is used as the backbone of realtime data stream, including online training, data integration, realtime data processing, service asynchronous interaction processing and cache data synchronization."
```
Comment (vvcephei) [inline powered-by.html]:
```suggestion
        "description": "At kuaishou, Kafka is used as the backbone of realtime data streams, including online training, data integration, realtime data processing, service asynchronous interaction processing and cache data synchronization."
```

File: powered-by.html [hunk 5]
```diff
@@ -237,6 +292,26 @@
...
+    }, {
+        "link": "https://www.laredoute-corporate.com",
+        "logo": "LaRedoute.svg",
+        "logoBgColor": "#FFFFFF",
+        "description": "La Redoute, the digital platform for families, uses Kafka as a central nervous system to decouple its application through business events. Thus, enabling a decentralized event-driven architecture bringing near-real-time data reporting, analytics and emerging AI-pipelines combining Kafka Connect, Kafka Streams and KSQL."
```
Comment (vvcephei) [inline powered-by.html]:
```suggestion
        "description": "La Redoute, the digital platform for families, uses Kafka as a central nervous system to decouple its application through business events. It enables a decentralized, event-driven architecture bringing near-real-time data reporting, analytics and emerging AI-pipelines combining Kafka Connect, Kafka Streams and KSQL."
```

**Review by vvcephei — COMMENTED** [review]
Hey @scott-confluent , thanks for this PR.

I'm assuming you've already rendered the page and made sure everything shows up nicely?

I had a couple of small copy-edit comments for you to consider.

**Comment by scott-confluent** [comment]
@vvcephei yep, I've rendered the page. happy to provide a screenshot of the updates

**Review by vvcephei — APPROVED** [review]
Thanks!

**Pull request merged**